colcon_sanitizer_reports/event_handlers/sanitizer_report.py

```diff
@@ -0,0 +1,41 @@
+# Copyright 2019 Open Source Robotics Foundation
+# Licensed under the Apache License, version 2.0
+
+from colcon_core.event.job import JobEnded
+from colcon_core.event_handler import EventHandlerExtensionPoint
+from colcon_core.location import get_log_path
+from colcon_core.plugin_system import satisfies_version
+from colcon_output.event_handler.log import STDOUT_STDERR_LOG_FILENAME
+from colcon_sanitizer_reports.sanitizer_log_parser import SanitizerLogParser
+
+
+class SanitizerReportEventHandler(EventHandlerExtensionPoint):
+    """Generate a report of all Sanitizer ERRORs and WARNINGs."""
+
+    ENABLED_BY_DEFAULT = False
```

Comment:
_ENABLED_BY_DEFAULT? (if we don't need this to be accessible outside the class)

Reply:
I don't know that there's any detriment to exposing the constant. This pattern is used in other colcon event handlers: https://github.com/colcon/colcon-output/blob/ad340e7c70e2c72a018dc483c209a6ab8c2db58b/colcon_output/event_handler/console_cohesion.py#L23-L38

Reply:
SGTM